tempesta_fw/t/functional/mixed_requests/mixed_test.py

```diff
@@ -7,6 +7,12 @@ class MixedRequests(stress.StressTest):
     script = None
     clients_num = int(tf_cfg.cfg.get('General', 'concurrent_connections'))
 
+    def create_servers(self):
+        port = tempesta.upstream_port_start_from()
+        server = control.Nginx(listen_port=port)
+        server.config.set_return_code()
+        self.servers = [server]
```

Comment:
Better to call `create_servers()` of the base class and then  set return code for each server: if base class function will changed, you won't need to update this test.

Reply:
fixed